File: q2.c
/* a simple program that makes a tuple of 5 fruits, makes a copy, and adds another fruit to it */
#include <stdio.h>
#include <string.h>
int main()
{
    char s[50][50], c[50][50];
    int i=0;
    printf("enter name of 5 fruits \n");
    for(i=0;i<5;i++){
        scanf("%s",s[i]);
        strcpy(c[i],s[i]);
        
    }
    printf("the list of fruits is \t");
    for(i=0;i<5;i++){
        printf("%s\t",s[i]);
    }
    printf("\nAdd the 6 th fruit to the new list\n");
    scanf("%s",c[i]);
    printf("the new list is\n");
    for(i=0;i<6;i++){
        printf("%s\t",c[i]);
    }

    return 0;
};

File: q2.py
#Making a tuple of 5 fruits, making a copy, and adding another fruit to it

fruits=[]
for i in range(5):
    fruit=input("enter fruit ")
    fruits.append(fruit)

copy_fruits = list(fruits)
new_fruit = input("Enter the 6th fruit to add: ")
copy_fruits.append(new_fruit)
print("The new list of fruits is", copy_fruits)
